src/main/cliapp/src/service/useSessionStorage.js

```diff
@@ -1,9 +1,10 @@
 import { useState, useEffect } from "react";
+import { LoggedInPersonService } from './LoggedInPersonService';
```

Comment:
@adamgibs can we get rid of useSessionStorage? Or do you think we will use it in the future... I would rather that we incorporate it into the useGetUserSettings... that way it would be much faster to pull the value from localStorage vs the DB.